freqtrade/persistence/migrations.py

```diff
@@ -92,7 +91,7 @@ def migrate_trades_table(decl_base, inspector, engine, table_back_name: str, col
             stoploss_order_id, stoploss_last_update,
             max_rate, min_rate, sell_reason, sell_order_status, strategy, buy_tag,
             timeframe, open_trade_value, close_profit_abs,
-            leverage, interest_rate, isolated_liq, is_short, interest_mode
+            leverage, interest_rate, isolated_liq, is_short
```

Comment:
you'll want to have `trading_mode` here instead ... 
at

Reply:
Sorry, the trading mode addition shouldn't be in this PR. I'm actually going to split this into 2 PRs, one for liq, and one for interest